### PR TITLE
Timeline 22.1

### PR DESCRIPTION
**Pull request by ghost**
Added new model for Airport (model for service. Which will be send air conditions). Was add Timeline Initializer + defalut model data. Remove some labels of form (only my views).

### CODE REVIEW

File: UtopiaCity/Common/DbInitializer.cs [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ﻿using System.Collections.Generic;
 using UtopiaCity.Common.Initializers;
-using UtopiaCity.Common.Interfaces;
```
Comment (ghost) [inline UtopiaCity/Common/DbInitializer.cs]: remove using UtopiaCity.Common.Interfaces;

**Review by ghost — COMMENTED** [review]
using UtopiaCity.Common.Interfaces нужна ли?

**Review by Ibraga96 — APPROVED** [review]

File: UtopiaCity/Common/DbInitializer.cs (hunk 1 above)
Comment (Ibraga96) [reply UtopiaCity/Common/DbInitializer.cs]: did you register your subDbInitializer? we using Interfaces namespace in the DbInitializer for this reason

Comment (Ibraga96) [reply UtopiaCity/Common/DbInitializer.cs]: and after removing that did you launch the project?

**Review by Ibraga96 — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
rewrite package versions

File: UtopiaCity/Common/DbInitializer.cs (hunk 1 above)
Comment (ghost) [reply UtopiaCity/Common/DbInitializer.cs]: Yes , I do . All working with out using UtopiaCity.Common.Interfaces;

**Review by ghost — COMMENTED** [review]
Remove using UtopiaCity.Common.Interfaces;

File: UtopiaCity/Common/DbInitializer.cs (hunk 1 above)
Comment (ghost) [reply UtopiaCity/Common/DbInitializer.cs]: image

File: UtopiaCity/Common/ISubDbInitializer.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using UtopiaCity.Data;
+
+namespace UtopiaCity.Common
```
Comment (Ibraga96) [inline UtopiaCity/Common/ISubDbInitializer.cs]: Mmm I see in your version ISubDbInitializer.cs file placed in UtopiaCity.Common directory thats'why you dont have to use this namespace "using UtopiaCity.Common.Interfaces", don't know if its right or not but Jenya will check

**Review by Ibraga96 — APPROVED** [review]

File: UtopiaCity/Common/ISubDbInitializer.cs (hunk 2 above)
Comment (Ibraga96) [reply UtopiaCity/Common/ISubDbInitializer.cs]: туц
Oo I just noticed )

Comment (ghost) [reply UtopiaCity/Common/ISubDbInitializer.cs]: You are right . Thank you)

Comment (Ibraga96) [reply UtopiaCity/Common/ISubDbInitializer.cs]: No prob)

**Review by Ibraga96 — APPROVED** [review]
Looks good!

File: UtopiaCity/Models/TimelineModel/PermitedModel.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UtopiaCity.Models.TimelineModel
+{
+    public class PermitedModel
+    {
+
```
Comment (crocomoth) [inline UtopiaCity/Models/TimelineModel/PermitedModel.cs]: Empty lines

File: UtopiaCity/Models/TimelineModel/ScheduleModel.cs [hunk 4]
```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UtopiaCity.Models.TimelineModel
+{
+    public class ScheduleModel
+    {
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+
```
Comment (crocomoth) [inline UtopiaCity/Models/TimelineModel/ScheduleModel.cs:10]: Attribute should be without empty lines between it and related code

**Review by crocomoth — APPROVED** [review]

File: UtopiaCity/Models/TimelineModel/PermitedModel.cs (hunk 3 above)
Comment (ghost) [reply UtopiaCity/Models/TimelineModel/PermitedModel.cs]: Thank you. I removed empty line

**Pull request merged**